imports/ui/components/setup/dropdown/AcadCohortDropdown.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Select from 'react-select';
+
+/*
+Temporary course option.
+@todo: Transfer Course array into another admin database
+*/
+const AcadCohort =[
```

Comment:
Must remember to incorporate these hard-coded values into logic itself and not leave them as code here! @bsweeyee  @IamVacer